starter_code/views/index.hbs

```diff
@@ -1,3 +1,14 @@
 <div class="index-container">
-  
+
+  <div class="beer-image">
+
+    <img src="/images/beer.png" alt="">
+
+    <a class="btn" href="/beers">Check the beers!</a>
```

Comment:
you could have used a more specific tag here to get the links `.beer-image a {...} ` rather than adding an extra class